<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnaPencere</class>
 <widget class="QWidget" name="AnaPencere">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>291</height>
   </rect>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>221</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>A Kenarı</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditA"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>B Kenarı</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditB"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonHesapla">
      <property name="text">
       <string>Hesapla</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Alan</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditAlan"/>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Çevre</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditCevre"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
